### ISSUE TITLE
Connect biolink terms directly to instances

### ISSUE THREAD

**Issue by balhoff** [TranslatorIIPrototypes/cam-pipeline#7]
The indirection between biolink terms and instance type assertions slows down queries. We should add axioms or rules to directly connect instances to their biolink types.

**Comment by balhoff** [comment]
@patrickkwang there are two ways this can be done:

1. Whenever an ontology term is mapped to a biolink term, make the ontology term a subclass of the biolink term. This will automatically make the reasoner create `rdf:type` links from instances to biolink terms.
2. Whenever an ontology term is mapped to a biolink term, create a SWRL rule which causes the reasoner to link individuals to biolink terms with some other custom relation.

So basically my question is, do you want to distinguish connections from instances to biolink terms from other ontology types they have (do we need to use a different relation)?
